src/test/java/homeworks/MyBaseTest.java

```diff
@@ -0,0 +1,87 @@
+package homeworks;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
+
+import java.time.Duration;
+
+public class MyBaseTest {
+
+    public WebDriver driver;
+    Actions actions;
+    String url;
+    public WebDriverWait wait;
```

Comment:
**I recommend:** make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
  protected WebDriver driver;
  protected WebDriverWait wait;
  protected Actions actions;
  protected String url;
```